### PR TITLE
Docker Images Comment in Docs

### PR DESCRIPTION
**Pull request by alonamid**
**Related issue**: #857 

**Type of change**: bug fix | new feature | other enhancement

**Impact**: other

**Release Notes**

### CODE REVIEW

**Comment by abejgonzalez** [comment]
This is failing the docs check.

File: docs/Chipyard-Basics/Initial-Repo-Setup.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ In CentOS-based platforms, we recommend installing the following dependencies:
 .. include:: /../scripts/centos-req.sh
    :code: bash
 
-In Ubuntu/Debian-based platforms (Ubuntu), we recommend installing the following dependencies:
+In Ubuntu/Debian-based platforms (Ubuntu), we recommend installing the following dependencies. These dependancies were written based on Ubuntu 16.04 LTS and 18.04 LTS - If they don't work for you, you can try out the Docker image (:ref:`Pre-built Docker Image`) before manually installing or removing dependancies:
```
Comment (tymcauley) [inline docs/Chipyard-Basics/Initial-Repo-Setup.rst:19]:
```suggestion
In Ubuntu/Debian-based platforms (Ubuntu), we recommend installing the following dependencies. These dependencies were written based on Ubuntu 16.04 LTS and 18.04 LTS - If they don't work for you, you can try out the Docker image (:ref:`Pre-built Docker Image`) before manually installing or removing dependencies:
```

Comment (abejgonzalez) [reply docs/Chipyard-Basics/Initial-Repo-Setup.rst]: Fix in separate commit.

**Review by abejgonzalez — APPROVED** [review]
LGTM

**Comment by abejgonzalez** [comment]
Documentation passes. Merging.

**Pull request merged**